### DISCUSSION TITLE
Ender 3 v2 with Sprite Pro and BL Touch

### DISCUSSION THREAD

**Discussion by jacauc** [mriscoc/Ender3V2S1#487] [Q&A] [answered]
I have an ender 3 v2 with a Sprite Pro hotend/extruder, BL touch and dual Z axis screws. Board 4.2.2
Is this firmware compatible, and if so, should I use the Ender 3 S1 firmware seeing that the v2 with all upgrades is almost like an S1?

Thanks in advance

**Comment by mriscoc** [comment, marked as answer]
Use the T13 version of V2 422 in https://github.com/mriscoc/Special_Configurations/releases/T13

**Reply by jacauc** [reply]
Thank you

**Reply by ozdenfurkan1** [reply]
is it last for now?

**Reply by lesgodadss** [reply]
hi, I have an E3 V2 422 with sprite pro cr touch and fine filament sensor but in your link I don't know which download I should take I'm a little lost in Firmware and can know github at the moment I I only have the creality officials and in your link there is in the assets this one: "Ender3V2-422-BLTUBL-T13-MPC-20240127.bin" is this one for my printer?

**Reply by ghost** [reply]
Yes that should be the one. I'm currently trying to get the same setup to work (Ender 3 v2, Sprite Pro and BLTouch) But there are some issues in this firmware that i can't seem to fix. My Sprite Pro doesnt recognize my BLTouch when plugged in directly, and the Sprite keeps ramming into the right side of the X-axis when auto-leveling (Looks like it thinks it's bigger then it actually is).

Still working on fixing those issues. Have you managed to get it working yourself?

-Matt

**Reply by lesgodadss** [reply]
no, I had a lot of difficulty with the settings each time I launched a print, the nozzle scratched the plate (probably because of the cr touch) so I put the frimwire back on origin of the ender and I do everything manually without the cr

**Reply by Koky05** [reply]
In Friday I install on mine Ender 3 V2 4.2.2 Sprite Pro Extruder with 3dTouch connected directly to it. I compile myself FW because I want IS, LA and Creality Cloud support. After change of physical settings according Extruder screw limitations and change Z-offset it works without any problems.

**Comment by NobiousDev** [comment]
Hello Miguel,
My setup is... _CR-10 frame, 4.2.7 board, Sprite Pro extruder, BL-touch, and Ender 3V2 display_. This setup is **new**. 
I installed your Ender 3 V2 427...T13 firmware and set all parameters and performed bed and nozzle PID tuning.
However, I constantly get an issue... after a couple of minutes or even during heating, the nozzle temperature drops by 3-4 degrees, and the thermal runaway alarm blocks the printer. 

Any suggestions?

**Reply by mriscoc** [reply]
Does your printer have the silicon jacket in the heater block?

**Reply by NobiousDev** [reply]
Yes, it has.

**Reply by NobiousDev** [reply]
I've just found _Experimental-Ender3V2-427-BLTUBL-LA-MPC-20221222.bin_. 
Can I use it with SpritEx Pro?

**Reply by mriscoc** [reply]
For Sprite extruder pro you need to use a T13 (Marlin thermistor calibration table 13) version.

**Reply by NobiousDev** [reply]
OK Thnx. By the way, I like your professional FW for E3 V2... !!

**Reply by datJok** [reply]
Hi miguel for the life of me i cant seem to find  the t13 file for the 427 board could you please share link thank you in advance

**Reply by mriscoc** [reply]
Hi. It is not possible for me to build all possible combinations for free. If you not found your specific pre-compiled binary firmware you need to compile your custom firmware using the sources in the repository: https://github.com/mriscoc/Special_Configurations#special-configurations

**Comment by kjdghcmdj** [comment]
Hello, I have an Ender 3v2 422 board with a cr touch, and I want to upgrade to a sprite pro extruder, I do not have dual z-axis motors or filament censor, it is just stock besides the cr touch. Would I use the T13 version firmware? And does that fix the issue I've seen online where the CR touch doesn't level properly?
